roles/lvm/tasks/debug.yml

```diff
@@ -0,0 +1,33 @@
+# This rules executed only when debug_storage_setup == true
```

Comment:
Which shouldn't happen in production. Should we remove this from the PR as well (since this is already well tested)?